main.go

```diff
@@ -24,34 +25,60 @@ func main() {
 	flag.Parse()
 
 	// check backend mode
-	if *modePtr == "serve" {
+	if *modePtr == "serve" { // HTTP serve mode
+
 		port := ":" + strconv.Itoa(*portPtr)
 		fs := http.FileServer(http.Dir("./build"))
 		http.Handle("/", fs)
 		log.Println("Listening on " + port + "...")
 		http.HandleFunc("/simulate", func(w http.ResponseWriter, r *http.Request) {
-			// TODO add simulation end-point handling and parsing parameters from body json
-			fmt.Println("Will add simulation endpoint soon!")
+
+			//load the parameters from HTTP
+			parameters, err := config.LoadParamFromHTTPRequest(r)
+			if err != nil {
+				http.Error(w, err.Error(), http.StatusBadRequest)
+				return
+			}
+
+			logFileName := runNewSimulation(parameters) //the logFileName is output in HTTP response to be used in dashboard
```

Comment:
```suggestion
			// logFileName returned to be used in dashboard
			logfileName := runNewSimulation(parameters)
```

Reply:
Use consistent capitalisation - you change it later. Might cause an issue down the line.